ext/spl/spl_dllist.stub.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+class SplQueue extends SplDoublyLinkedList
+{
+    /**
+     * @param mixed $value
+     * @return bool
```

Comment:
As it always returns `true`, I think this could be converted to `void`.

Reply:
Yeah, let's work on https://bugs.php.net/55173 and https://bugs.php.net/75958